gitlab/objects.py

```diff
@@ -1253,6 +1253,18 @@ class ProjectKey(GitlabObject):
 class ProjectKeyManager(BaseManager):
     obj_cls = ProjectKey
 
+    def enable_deploy_key(self, project_id, key_id):
+        """Enable a deploy key in a project."""
+        url = '/projects/%s/deploy_keys/%s/enable' % (project_id, key_id)
+        r = self.gitlab._raw_post(url)
+        raise_error_from_response(r, GitlabBuildRetryError, 201)
```

Comment:
Also I'd rather add ``enable`` and ``disable`` methods on the ``ProjectKey`` class instead of adding these methods on the manager, to keep the existing logic.